examples/src/main/java/ExampleVerifyEmail.java

```diff
@@ -0,0 +1,49 @@
+import com.messagebird.MessageBirdClient;
+import com.messagebird.MessageBirdService;
+import com.messagebird.MessageBirdServiceImpl;
+import com.messagebird.exceptions.GeneralException;
+import com.messagebird.exceptions.NotFoundException;
+import com.messagebird.exceptions.UnauthorizedException;
+import com.messagebird.objects.Verify;
+import com.messagebird.objects.VerifyMessage;
+import com.messagebird.objects.VerifyRequest;
+
+/**
+ * Created by faizan on 10/12/15.
+ */
+public class ExampleVerifyEmail {
+
+    public static void main(String[] args) throws UnauthorizedException, GeneralException, NotFoundException {
+    	
+    	final String ACCESS_KEY = args[0];
```

Comment:
as they were all arguments I though it would make it easier for the reader to understand it. And indeed as they could also be seen as "constants" for that given execution.

Reply:
I also would prefer camelCase but this is an example class to show how customers can use the endpoints and testing, I am okay with upper-case.